src/ontology/config/prefixes.csv

```diff
@@ -224,8 +224,15 @@ PLANP,http://purl.obolibrary.org/obo/PLANP_
 CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
+ICD10CM2,https://icd.codes/icd10cm/
```

Comment:
## ICD10CM
<details><summary>Details</summary>
<p>

I don't remember where, but I saw 2 different valid variations, so I added the missing one:
```csv
ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
ICD10CM2,https://icd.codes/icd10cm/
```

</p>
</details>